static/scss/auth.scss

```diff
@@ -242,3 +242,8 @@
   width: 50px;
   height: 50px;
 }
+
+.policy-consent {
+  font-size: 14px;
```

Comment:
I don't think this needs to be smaller than the body text.

Reply:
Resolved.